### ISSUE TITLE
Cannot regenerate ER diagram

### ISSUE THREAD

**Issue by MangoMango666** [dbeaver/dbeaver#3316]
Hello.

I am using DBeaver 5.0.1 Community edition on a Linux Mint computer.
It is connected to a Postgresql 10 database.
I have created an ER diagram from the database.
However, when I try to regenerate the ER diagram (right-click on the ER diagram > regenerate), it does not seem to work (I do changes manually in the db but the ER diagram is unchanged).
I don't get any error message.

The only work-around I found is to delete the existing ER diagram and create a new one.

Is there anything I am missing ?

**Comment by serge-rider** [comment]
There is not such command as "regenerate".
There are two other commands:
- Arrange diagram: auto-layout tables
- Refresh diagram: refreshes tables from the database (works only for table/schema diagrams and doesn't work for custom ones)

**Comment by MangoMango666** [comment]
Hello and thanks for the help.

I wrote 'regenerate' because it would be the translation from the word used in the GUI in my language (French ; it is written 'régénérer'). I guess the proper translation should be 'refresh'. Actually, I had found myself curious it was not written 'refresh'. I may post a screen capture if needed.

Still, it does not seem to work. The ER diagram is a standard one, as far as I understand, generated by DBeaver in the first place and not manually modified.
